Pokepedia.Domain/Services/Crud/Read/IReadByNameServiceForLocation.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Pokepedia.Domain.Services.Crud.Read
+{
+    public interface IReadByNameServiceForLocation<T, TResult>
```

Comment:
Service in ANY file name or class name should be the postfix. So it should be IReadByNameForLocationService